data/campaigns/The_South_Guard/scenarios/06a_Into_the_Depths.cfg

```
@@ -275,40 +275,20 @@
 #ifdef EASY
         {GENERIC_UNIT 2 (Revenant) 19 26} {GUARDIAN}
         {GENERIC_UNIT 2 (Revenant) 19 29} {GUARDIAN}
-        {NAMED_GENERIC_UNIT 2 (Dark Sorcerer) 20 27 (Mal Maul) (_"Mal Maul")} {GUARDIAN}
+        # po: Dark Sorcerer name
+        {NAMED_GENERIC_UNIT 2 (Dark Sorcerer) 20 27 (Mal Maul) (_"Mal Maul") FEMALE_NAME=_"female^Mal Maul"} {GUARDIAN}
```

Does this po note get applied to both the male and female versions of the string? If not, the first of these probably needs to be split into multiple lines, something like this:

```suggestion
        # po: Dark Sorcerer name
        {NAMED_GENERIC_UNIT 2 (Dark Sorcerer) 20 27 (Mal Maul) (_"Mal Maul")
        # po: Dark Sorceress name
        FEMALE_NAME=_"female^Mal Maul"} {GUARDIAN}
```